server/config.go

```diff
@@ -85,6 +88,9 @@ type Config struct {
 	electionMs uint64
 
 	configFile string
+
+	// For all warnings during parsing.
+	WarningMsgs []string
```

Comment:
Exporting WarningMsg is too, what is the WarningMsgs for? what I first see is that there is a WarningMsg config.